src/mSupplyMobileApp.js

```diff
@@ -205,6 +212,15 @@ class MSupplyMobileAppContainer extends React.Component {
           >
             <SupplierCredit />
           </ModalContainer>
+
+          <ModalContainer
+            isVisible={isBreachModalOpen}
+            onClose={closeBreachModal}
+            title={breachModalTitle}
+            fullScreen
+          >
+            <BreachDisplay />
+          </ModalContainer>
```

Comment:
keeping this breach modal at the root level as it'll be opened from customer invoice, vaccine page and current stock page